flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -77,7 +77,7 @@ static constexpr CategorySet AnyType{IntrinsicType | DerivedType};
 
 ENUM_CLASS(KindCode, none, defaultIntegerKind,
     defaultRealKind, // is also the default COMPLEX kind
-    doublePrecision, defaultCharKind, defaultLogicalKind,
+    doublePrecision, defaultCharKind, defaultLogicalKind, greaterAndEqualToKind,
```

Comment:
This name is nonsensical.  How can a value be both greater than and equal to another?

Reply:
would `noLessThanKind` be a better name?

Reply:
`greaterOrEqualsKind` is a possibility.

Reply:
Sorry I was blind... Test file `execute_command_line.f90` has been added to `flang/test/Semantics/` testing for errors when less than, no errors when greater than or equal to.